src/Symfony/Component/Translation/Bridge/Loco/Tests/LocoProviderFactoryWithoutTranslatorBagTest.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Symfony\Component\Translation\Bridge\Loco\Tests;
+
+use Symfony\Bridge\PhpUnit\ExpectDeprecationTrait;
+use Symfony\Component\Translation\Bridge\Loco\LocoProviderFactory;
+use Symfony\Component\Translation\Provider\ProviderFactoryInterface;
+
+/**
+ * This test should be removed in Symfony 7.0 when making $translatorBag argument not nullable for the "LocoProviderFactory" constructor.
```

Comment:
`@group legacy` says it already
but why do we need the deprecation in the first place?

Reply:
See https://github.com/symfony/symfony/pull/44484#discussion_r773176509 to know why I've added a deprecation (in LocoProvider and its factory)

Reply:
please remove it, it's not needed for sure